examples/time/time.bal

```diff
@@ -34,12 +34,12 @@ public function main() returns error? {
     time:Seconds utcDiff = time:utcDiffSeconds(currentUtc, utc1);
     io:println(`UTC diff: ${utcDiff}`);
 
-    // Check the given `time:Date` value is valid or not.
+    // Checks the given `time:Date` value is valid or not.
     time:Date date = {year: 1994, month: 11, day: 7};
     boolean isValidDate = (time:dateValidate(date) is ()) ? true : false;
     io:println(string `Is valid date: ${isValidDate}`);
 
-    // Return the day of week as a number starting from 0 to 6 inclusive.
+    // Returns the day of week as a number starting from 0 to 6 inclusive.
```

Comment:
```suggestion
    // Returns the day of the week as a number starting from 0 to 6 inclusive.
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383